build.rs

```diff
@@ -1,18 +1,59 @@
+use dotenv::dotenv;
+use serde::Deserialize;
+use std::env;
+use std::io::{prelude::*, Cursor};
+
+#[allow(dead_code)]
+#[derive(Deserialize, Clone)]
+struct WorkFlowRun {
```

Comment:
This is fine, but could also consider using something like [this](https://crates.io/crates/github) at some point - especially if there's a crate with official support.

Should add a comment that these structs are github api responses.

Reply:
I'll add a comment about it. For two HTTP requests I'm not sure I'd want to bring in an entire dependency, but maybe there's a chance we'll want to do more sophisticated stuff.